src/GrassPokemon.java

```diff
@@ -0,0 +1,35 @@
+public class GrassPokemon extends Pokemon {
+
+    private int grassHeight = 1;        // centimeters
+    private int grassCuttingHeight;
+
```

Comment:
Mooi gebruik gemaakt van verschillende variabele in je subklasse. Doordat deze verschillen heeft je subklasse bestaansrecht en staan de instance variabelen op het juiste niveau